### DISCUSSION TITLE
https://blog.tidy-intelligence.com/posts/sakura-visualizations/

### DISCUSSION THREAD

**Discussion by giscus** [tidy-intelligence/blog#5] [Announcements]
# https://blog.tidy-intelligence.com/posts/sakura-visualizations/

Visualizing first and full cherry blossom bloom dates for Japanese regions using R and Python

https://blog.tidy-intelligence.com/posts/sakura-visualizations/

**Comment by joethesaint** [comment]
Interesting read. Did you use a notebook such as ipynb? And what’s the equivalent in Rust

**Reply by christophscheuch** [reply]
I just used Quarto to render the R and Python code chunks ✌️ 

No idea about Rust though, I have never programmed in it..

**Comment by mitchhenderson** [comment]
Thanks for this post, very useful. Big fan of how you're able to show R and Python code together like this. Can you recommend any resources where I can learn how to present code like that?

**Reply by christophscheuch** [reply]
Thank you for the kind words! I'm just using Quarto tabsets (see https://quarto.org/docs/interactive/layout.html) and you could check out one of my blog posts on GitHub (https://github.com/tidy-intelligence/blog/blob/main/posts/sakura-visualizations/index.qmd). There is also a video recording of me from a Berlin R Users Group meetup where I talk about the multi-language setup starting around here: https://youtu.be/Sm7BpxhUMZo?si=_auSiqgXeIFD6AFb&t=1507

**Reply by mitchhenderson** [reply]
Thanks!
